packages/labs/context/src/lib/std-decorators/consume.ts

```diff
@@ -0,0 +1,68 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {ReactiveElement} from '@lit/reactive-element';
+import {ContextConsumer} from '../controllers/context-consumer.js';
+import {Context} from '../create-context.js';
+
+/**
+ * A property decorator that adds a ContextConsumer controller to the component
```

Comment:
Do we want to get more specific with the decorator type? "A field and accessor decorator" maybe?